_config.yml

```diff
@@ -49,6 +49,8 @@ contentful:
     query: fields.category=common
     map:
       content_block: content
+  page:
+    query: "fields.collections[exists]=false"
```

Comment:
FYI @j-brammer This update relies on migrations introduced via crdschurch/crds-contentful-migrations#157

Reply:
Created demo PR for contentful-migrations 
https://github.com/crdschurch/crds-contentful-migrations/pull/158